src/data/programs.ts

```diff
@@ -26,42 +26,76 @@ const ira_programs = {
       en: 'Federal Alternative Fuel Vehicle Refueling Property Credit (30C)',
       es: 'Crédito a la propiedad para el repostaje de vehículos de combustible alternativo (30C)',
     },
+    url: {
+      en: 'https://www.irs.gov/credits-deductions/alternative-fuel-vehicle-refueling-property-credit',
+      es: 'https://www.irs.gov/es/credits-deductions/alternative-fuel-vehicle-refueling-property-credit',
+    },
   },
   cleanVehicleCredit: {
     name: {
       en: 'Federal Clean Vehicle Credit (30D)',
       es: 'Crédito para vehículos limpios (30D)',
     },
+    url: {
+      en: 'https://www.irs.gov/credits-deductions/credits-for-new-clean-vehicles-purchased-in-2023-or-after',
+      es: 'https://www.irs.gov/es/credits-deductions/credits-for-new-clean-vehicles-purchased-in-2023-or-after',
+    },
   },
   creditForPreviouslyOwnedCleanVehicles: {
     name: {
       en: 'Federal Credit for Previously-Owned Clean Vehicles (25E)',
       es: 'Crédito para vehículos limpios de propiedad anterior (25E)',
     },
+    url: {
+      en: 'https://www.irs.gov/credits-deductions/used-clean-vehicle-credit',
+      // TODO this page's content isn't translated into Spanish, but this is
```

Comment:
looks translated to me?

Reply:
Oh wow, they definitely just published that in the last few days.